api/restapi/configure_keptn.go

```diff
@@ -31,6 +32,7 @@ import (
 const envVarLogLevel = "LOG_LEVEL"
 
 type EnvConfig struct {
+	AuthEnabled           bool    `envconfig:"MAX_AUTH_ENABLED" default:"true"`
```

Comment:
let's also change the name of the variable to `MaxAuthEnabled`, otherwise the name of the variable would suggest that it's used to disable authentication in general

Reply:
I prefer keeping the same naming like the other env vars